pom.xml

```diff
@@ -77,7 +77,7 @@
         <thrift.version>0.9.1</thrift.version>
         <hadoop.version>2.5.2</hadoop.version>
         <spark.version>2.0.0</spark.version>
-        <hbase.version>1.2.3</hbase.version>
+        <hbase.version>1.3.1</hbase.version>
```

Comment:
this will probably require cqs... maybe we should have this as a separate pr, and/or hold off on bumping hbase versions until GM 1.4?

Reply:
I'd suggest CQs 'now', and hold the HBase updates until 1.4.0.  

Tentatively, we could start a '1.4.x' CQ-loose-kicking PR.